### DISCUSSION TITLE
키퍼 간단한 Slack 퍼블봇을 만들어보자

### DISCUSSION THREAD

**Discussion by 02ggang9** [BDD-CLUB/challenge-every-week-posting#156] [1. 개발 관련 글 모음]
https://stump-cress-2f5.notion.site/Slack-7741a8de99274f199c9624a305bae127?pvs=4

간단한 Slack 퍼블봇을 만들어봤습니다~!
다음은 가능할지는 모르겠지만 키퍼 R2에서 쓰이고 있는 @Member 어노테이션에 관한 글을 써보고 싶네요..

겨울방학 Day - 113 ..ㅠㅠ

**Comment by shkisme** [comment]
우왕...!!! 이거 바로 적용해도 괜찮겠는데요?? 대박 👍

**Reply by 02ggang9** [reply]
굳굳 👍
